generators/server/templates/src/main/java/package/config/_UaaConfiguration.java

```diff
@@ -87,9 +91,8 @@ public void configure(ClientDetailsServiceConfigurer clients) throws Exception {
          */
         clients.inMemory()
             .withClient("web_app")
-            .scopes("web-app")
+            .scopes("openid")
```

Comment:
ugh.....i read the false line! I thought you renamed the clientId...

well, i still wouldn't use "openid" as scope, because this actually has not much to do with openid....we do it like openid, without strict openID....but this are nerdy details not really changing the working behaviour...